### ISSUE TITLE
Ship Acceleration

### ISSUE THREAD

**Issue by Corey56** [Corey56/RingLeader#1]
The ship has no max speed. Do we think it should?

**Comment by ian-thomas-41** [comment]
Yes! From personal experience, I believe that the ship should not constantly accelerate as this tends to fling the ship into the bubbles and results in an angry user.
